sound/pci/hda/hda_codec.c

```diff
@@ -925,8 +925,28 @@ snd_hda_codec_device_init(struct hda_bus *bus, unsigned int codec_addr,
 	}
 
 	codec->bus = bus;
+	codec->depop_delay = -1;
```

Comment:
typo in commit: s/Iinitialization/Initialization/

Reply:
Should I send an update here or can this be done when sending series to upstream?

Reply:
@crojewsk-intel Can be done when sending upstream, no problem.